gitleaks.toml

```diff
@@ -0,0 +1,7 @@
+title = "Gitleaks config"
+
+[rules.allowlist]
```

Comment:
What if we generated the certificate/key during the test? It'd save us from another config file in our repos. 

We use https://github.com/madflojo/testcerts elsewhere